src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -16,13 +16,11 @@ public class DescriptionNameTests
         // When running both inner and outer loop together, dump only once
         private static bool s_dumpedRuntimeInfo = false;
 
-        private static readonly bool s_isInHelix = Environment.GetEnvironmentVariables().Keys.Cast<string>().Where(key => key.StartsWith("HELIX")).Any();
-
         [Fact]
         [SkipOnPlatform(TestPlatforms.Browser, "throws PNSE when binariesLocation is not an empty string.")]
         public void DumpRuntimeInformationToConsole()
         {
-            if (s_dumpedRuntimeInfo || !s_isInHelix)
+            if (s_dumpedRuntimeInfo || !PlatformDetection.IsInHelix)
```

Comment:
Ah I see the helix bool already existed. I think the new location is much better.